### PR TITLE
[TOPI][ADRENO] Add Group Conv2d texture schedule

### PR DESCRIPTION
**Pull request by krishnaraj36**
Implemented the adreno schedule for group conv2d op with texture enablement.

### CODE REVIEW

File: python/tvm/topi/adreno/group_conv2d_nchw.py [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+    )
+    cfg.define_split(
+        "tile_x",
+        x,
+        num_outputs=3,
+        filter=lambda entity: entity.size[1] <= 8 and entity.size[2] <= 16,
+    )
+
+    cfg.define_split("tile_rcc", rcc, num_outputs=2)
+    cfg.define_split("tile_ry", ry, num_outputs=2)
```
Comment (tqchen) [inline python/tvm/topi/adreno/group_conv2d_nchw.py:279]: Thanks @krishnaraj36 , as we are moving towards meta-schedule and relax, we are moving away from legacy pipelines in updated flows, would be great to bring the new schedules under TensorIR, and dlight pipeline instead

**Review by tqchen — CHANGES_REQUESTED** [review]

File: python/tvm/topi/adreno/group_conv2d_nchw.py (hunk 1 above)
Comment (krishnaraj36) [reply python/tvm/topi/adreno/group_conv2d_nchw.py:279]: @tqchen  - So that mean relay legacy pipeline will be deprecated ?
@srkreddy1238 FYI

Comment (tqchen) [reply python/tvm/topi/adreno/group_conv2d_nchw.py:279]: ideally we would like to move toward the relax-based pipeline,  that would help us to leverage the latest features also simplify some of the developments

**Review by tqchen — APPROVED** [review]

File: python/tvm/topi/adreno/group_conv2d_nchw.py (hunk 1 above)
Comment (tqchen) [reply python/tvm/topi/adreno/group_conv2d_nchw.py:279]: I am merging in this one for now, but would be good to do gthe transition for future ones :)

**Pull request merged**